hotel_elasticsearch/configuration.py

```diff
@@ -24,3 +24,27 @@ def save(self):
         with open(self.config_file, 'w') as f:
             config = yaml.safe_dump(dict(self), default_flow_style=False)
             f.write(config)
+
+
+class HotelElasticSearchConfig(OrderedDict):
+    def __init__(self, config_file="/etc/hotel-elasticsearch/hotel-config.yml"):
+        try:
+            with open(config_file) as f:
+                super(HotelElasticSearchConfig, self).__init__(sorted(yaml.safe_load(f.read()).items()))
+        except FileNotFoundError:
+            pass
```

Comment:
Maybe log some error that the config file wasn't found?

Reply:
Well it's implemented as optional, it will work without it. But maybe a warning